server/others/commands.lua

```diff
@@ -145,11 +140,6 @@ ATL.RegisterCommand('info', 'My character info', 'user', function(player)
   print(('Name: %s | Character ID: %s | Character Name: %s | Group: %s | Money: %s$ | Bank: %s$ | Job: %s - %s | On Duty: %s'):format(GetPlayerName(player.source), player:getCharacterId(), player:getCharacterName(), player:getGroup(), player:getAccount('cash'), player:getAccount('bank'), player:getJobLabel(), player:getRankLabel(), player:getDuty()))
 end, {}, false)
 
-ATL.RegisterCommand('coords', 'Get coords', 'admin', function(player)
```

Comment:
Why removing this? It would not be better to just refactor the command?

Reply:
Check the name of the commit name.

Reply:
Ignore the past comment. We will refactor a couple a command with the new command handler.